### DISCUSSION TITLE
Snapper Rollback - SpiralLinux_Gnome_12.231008_x86-64.iso - not booting into rollback image which has been made default.

### DISCUSSION THREAD

**Discussion by mmcdonald3009** [SpiralLinux/SpiralLinux-project#333] [Q&A] [answered]
HI Everyone,
I started using: SpiralLinux_Gnome_12.231008_x86-64.iso

After booting into a read-only snapshot. and invoking snapper rollback in a terminal then states it has created a read]write copy of specified ID and set it as the the default. However, it is not actually booting into the default. 

Happy to trouble shoot myself if someone could please advise where the script is located - or if you've come across this and can post the solution

**Comment by geckolinux** [comment, marked as answer]
Hi there, for SpiralLinux it's not `snapper rollback` but rather `snapper-rollback <num>` .
https://github.com/SpiralLinux/SpiralLinux-project/wiki#working-with-btrfs-snapshots-and-rollbacks

After having run `snapper rollback` I'm not sure exactly in what state the subvolumes will be or if they will work as designed for SpiralLinux with `snapper-rollback`, so if it's a new installation the fastest way to fix things would probably be re-installing.

**Reply by mmcdonald3009** [reply]
Thanks for all your efforts and yes, the syntax issue I missed - it works as supposed to snapper-rollback number
